<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QToolButton" name="btnSaveCache1">
          <property name="text">
           <string>W1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnLoadCache1">
          <property name="text">
           <string>R1</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QToolButton" name="btnSaveCache2">
          <property name="text">
           <string>W2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnLoadCache2">
          <property name="text">
           <string>R2</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QToolButton" name="btnSaveCache3">
          <property name="text">
           <string>W3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnLoadCache3">
          <property name="text">
           <string>R3</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QToolButton" name="btnSaveCache4">
          <property name="text">
           <string>W4</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnLoadCache4">
          <property name="text">
           <string>R4</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnZoomIn">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnZoomOut">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnZoomFit">
        <property name="text">
         <string>Fit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnScreenCapture">
        <property name="text">
         <string>Capture</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="btnOpen">
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnClear">
        <property name="text">
         <string>Clr</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnTest">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="GraphWidget" name="graphView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="cbxAutoFit">
        <property name="text">
         <string>&amp;Automatic fit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbxHighQuality">
        <property name="text">
         <string>High &amp;Quality</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbxBackground">
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbxSceneArea"/>
      </item>
      <item>
       <widget class="QTreeWidget" name="tblLayers">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustIgnored</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideNone</enum>
        </property>
        <property name="itemsExpandable">
         <bool>false</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="headerStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphWidget</class>
   <extends>QFrame</extends>
   <header>graphwidget.h</header>
   <container>1</container>
   <slots>
    <signal>onAutoFitChanged(bool)</signal>
    <signal>onBackgroundChanged(int)</signal>
    <signal>onHighQualityChanged(bool)</signal>
    <slot>setAutoFit(bool)</slot>
    <slot>setBackground(int)</slot>
    <slot>setHighQuality(bool)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>cbxAutoFit</sender>
   <signal>toggled(bool)</signal>
   <receiver>graphView</receiver>
   <slot>setAutoFit(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>789</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>624</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>graphView</sender>
   <signal>onAutoFitChanged(bool)</signal>
   <receiver>cbxAutoFit</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>550</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>789</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>graphView</sender>
   <signal>onBackgroundChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>handleBackgroundChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>558</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>758</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbxHighQuality</sender>
   <signal>toggled(bool)</signal>
   <receiver>graphView</receiver>
   <slot>setHighQuality(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>789</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>658</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>graphView</sender>
   <signal>onHighQualityChanged(bool)</signal>
   <receiver>cbxHighQuality</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>662</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>789</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>handleBackgroundChange(int)</slot>
  <slot>handleSceneAreaChange(int)</slot>
 </slots>
</ui>
